tests/test_trusted-execution.yml

```diff
@@ -0,0 +1,132 @@
+---
+- name: Basic sanity test for fapolicyd
+  hosts: localhost
```

Comment:
```suggestion
  hosts: all
```
the test framework will provide the correct hostname

Reply:
DONE.